tests/bsim/bluetooth/ll/cis/tests_scripts/connected_iso.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+# Copyright 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+# Basic Connected ISO test: a Central connects to 9 Peripherals and Establishes
+# 1 CIS each to 9 Peripherals (9 CIS in a CIG)
+simulation_id="connected_iso"
+verbosity_level=2
+process_ids=""; exit_code=0
+
+function Execute(){
+  if [ ! -f $1 ]; then
+    echo -e "  \e[91m`pwd`/`basename $1` cannot be found (did you forget to\
+ compile it?)\e[39m"
+    exit 1
+  fi
+  timeout 120 $@ & process_ids="$process_ids $!"
+}
```

Comment:
The babblesim test scripts were recently refactored to not define `Execute` themselves, but rather get it from a common file
```c
source ${ZEPHYR_BASE}/tests/bsim/sh_common.source
```

Reply:
Done